docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -15,6 +27,30 @@ TIP: To sort by any other column, switch on the *Technical preview* toggle above
 
 For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
 
+[float]
+[[rule-execution-logs]]
+=== Rule execution logs
+
+Each detection rule keeps a log of every time it ran, its success or failure, any warning or error messages, and how long it took to search for data, create alerts, and complete its execution. This can help you troubleshoot a particular rule if it isn't behaving as expected (for example, if it isn't creating alerts or takes a long time to run).
+
+To access a rule's execution log, go to *Detect* -> *Rules*, click the rule's name to open the rule details, then scroll down and select the *Rule execution logs* tab.
+
+[role="screenshot"]
+image::images/rule-execution-logs.png[Rule execution logs table]
+
+You can hover over each column heading to display a tooltip about that column's data. Click a column heading to sort the table by that column.
+
+Use these controls to narrow down what's included in the logs table:
+
+* The *Status* drop-down filters the table by rule execution status: 
+** *Succeeded*: The rule was able to complete its defined search. This doesn't necessarily mean it generated an alert, just that it ran without error.
+** *Failed*: The rule encountered an error that prevented it from running. For example, a machine learning rule whose corresponding machine learning job wasn't running.
+** *Partial failure*: Nothing prevented the rule from running, but it still might not have returned the expected results. For example, a custom query rule tried to search an index pattern that couldn't be found in {es}.
+
+* The date and time picker sets the time range of rule executions included in the table. This overrides the time range set at the top of the rule details page.
```

Comment:
> This overrides the time range set at the top of the rule details page.

I had to re-read this one, but you're just meaning that it is separate/independent of the global time picker, correct? If so, 👍.

Reply:
@spong You're correct, but if you had to re-read this maybe it wasn't phrased well. I like your term "separate" so I'll try that:
```suggestion
* The date and time picker sets the time range of rule executions included in the table. This is separate from the global date and time picker at the top of the rule details page.
```

Reply:
Sounds good - thanks Joe! 🙂